src/core/a-entity.js

```diff
@@ -290,19 +294,23 @@ var proto = Object.create(ANode.prototype, {
 
       // Check if component already initialized.
       if (name in this.components) { return; }
-      component = this.components[name] = new components[name].Component(this, data);
+      if (id && !components[name].multiple) {
+        throw new Error('Trying to initialize multiple components of type ' + name +
```

Comment:
can we indent like we do here: https://github.com/aframevr/aframe/pull/1596/files#diff-dc3043915bc0e1dc643b7fa488e01c0eR254